report_thesis/src/sections/related_work.tex

```diff
@@ -46,11 +49,78 @@ \section{Related Work}
 The authors demonstrate the effectiveness of a deep CNN in classifying geochemical standard samples using \gls{libs} spectra collected at varying distances.
 This addresses the challenge of spectral differences induced by distance, showcasing that \gls{cnn} can learn to classify samples without the need for traditional spectral preprocessing or distance correction.
 Using a dataset of over 18,000 \gls{libs} spectra from 39 geochemical standard samples, the study compares the \gls{cnn} model's performance against four other machine learning models: \gls{bpnn} \gls{svm}, \gls{lda}, and \gls{lr}.
-The CNN model exhibits superior classification accuracy, emphasizing its potential for geochemical sample identification/classification in planetary exploration.
+The \gls{cnn} model exhibits superior classification accuracy, emphasizing its potential for geochemical sample identification/classification in planetary exploration.
 The paper includes a detailed comparative analysis, proving the \gls{cnn} model's superior performance.
 With classification accuracies on the validation set for all models exceeding 95\%, the \gls{cnn} model demonstrated the highest overall accuracy.
 This was particularly evident as the training set size increased, indicating the model's robustness to varying distances without requiring distance correction.
 Statistical analysis further confirmed the \gls{cnn} model's superiority, with higher average Ncorr values compared to other models.
-The CNN model's ability to accurately classify geochemical samples without preprocessing for distance correction is quite impressive.
+The \gls{cnn} model's ability to accurately classify geochemical samples without preprocessing for distance correction is quite impressive.
 This is particularly relevant to our work because we are also working with \gls{libs} spectra collected at varying distances.
-The comparative analysis underscores the CNN model as a best-fit approach for \gls{libs} data analysis, potentially setting a new standard for future research and applications in the field.
\ No newline at end of file
+The comparative analysis underscores the \gls{cnn} model as a best-fit approach for \gls{libs} data analysis, potentially setting a new standard for future research and applications in the field.
+
+\citet{jeonEffectsFeatureEngineering2024} investigated the effects of feature engineering on the robustness of \gls{libs} for steel classification.
+They developed a remote \gls{libs} system to classify six steel types, using various feature-engineering and machine learning algorithms, including SVM and FCNN, to handle different laser energies in test datasets.
```

Comment:
```suggestion
They developed a remote \gls{libs} system to classify six steel types, using various feature-engineering and machine learning algorithms, including \gls{svm} and FCNN, to handle different laser energies in test datasets.
```
Need to add FCNN to glossary.

Reply:
Reminder to self: Fully-connected neural network (FCNN)